src/components/light.js

```diff
@@ -20,7 +20,21 @@ module.exports.Component = registerComponent('light', {
     intensity: {default: 1.0, min: 0, if: {type: ['ambient', 'directional', 'hemisphere', 'point', 'spot']}},
     penumbra: {default: 0, min: 0, max: 1, if: {type: ['spot']}},
     type: {default: 'directional', oneOf: ['ambient', 'directional', 'hemisphere', 'point', 'spot']},
-    target: {type: 'selector', if: {type: ['spot', 'directional']}}
+    target: {type: 'selector', if: {type: ['spot', 'directional']}},
+
+    // Shadows.
```

Comment:
Should we make the light a component with dynamic schema if a the property `castShadows: true`?